src/bin/command.ts

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
In general I don't love the `command.ts` file, why don't we use individual command files instead for `release-pr` and `github-release`, and then test these in isolation?

see: https://github.com/yargs/yargs/blob/master/docs/advanced.md#example-command-hierarchy-using-commanddir

Reply:
I find the commands/options configuration here confusing - @bcoe more a question for you:

many `release-pr` sub-options have the same name as `github-release` sub-options:
`release-type`, `package-name`, `repo-url`, `path`, `label`

`release-type` appears in both commands AND here

some options outside of a command are only relevant to a particular command:
`bump-minor-pre-major` and `release-as` belong only to `release-pr`

I haven't fully grokked yargs but it seems like we could clean this up a bit by deduplicating `release-type` (already done?), `package-name`, `repo-url`, `path`, and `label` (though maybe this last one is different for each command?) down into this common section and moving `bump-minor-pre-major` and `release-as` up into `release-pr`?

I like the addition of `.config()` but it also kind of further confuses me wrt to which options go with which command. Is any option from either command allowed in the config file json? if so, what happens the config has a `changelog-types` key (only defined on `release-pr`) and the `github-release` command is invoked?

Reply:
> I haven't fully grokked yargs but it seems like we could clean this up a bit by deduplicating release-type (already done?), package-name, repo-url, path, and label (though maybe this last one is different for each command?) down into this common section and moving bump-minor-pre-major and release-as up into release-pr?

This seems like it would be a smart cleanup, options can be defined at the top level, or at the command level in a `builder` function.